app.test.js

```diff
@@ -10,6 +10,70 @@ afterAll(() => {
 beforeEach(() => {
   return seed(testData);
 });
+  describe("/api/categories", () => {
+    it("GET 200: should respond with an array of category objects, each of which should have two properties: slugs and description ", () => {
+      return request(app)
+        .get("/api/categories")
+        .expect(200)
+        .then(({ body }) => {
+          const { categories } = body;
+          expect(categories).toBeInstanceOf(Array);
+          expect(categories).toHaveLength(4);
+          categories.forEach((category) => {
+            expect(category).toMatchObject({
+              slug: expect.any(String),
+              description: expect.any(String),
+            });
+          });
+        });
+    })
+    it("404: , responds with an error message when passed a non-existent path", () => {
+      return request(app) 
+        .get("/api/categoris")
+        .expect(404)
+        .then(({body}) => {
+          expect(body.msg).toBe("Page not found!");
+        });
+    });
+  })
+  describe('/api/reviews/:review_id', () => {
+    it('200: should respond with the review object of requested id. ', () => {
+        return request(app)
+        .get('/api/reviews/3')
+        .expect(200)
+        .then(({_body}) => {
+            const {review} = _body
+            expect(review).toMatchObject({
+              review_id: expect.any(Number),
```

Comment:
Need to check you're getting review_id 3, if you've asked for that one. 

at the moment, this would pass for any review sent back